src/store/engine.ts

```diff
@@ -28,6 +29,9 @@ export const engineStore = createPartialStore<EngineStoreTypes>({
         engineIds,
         engineInfos,
       });
+
+      // 代替ポート情報を取得 → トースト通知が発火する
+      dispatch("GET_ALT_PORT_INFOS");
```

Comment:
念のためにawaitしておいたほうが良いかも
```suggestion
      await dispatch("GET_ALT_PORT_INFOS");
```